docs/source/configuration.rst

```diff
@@ -99,4 +99,52 @@ RESPONSE_VALIDATION
     that they conform to the protocol. This should only be used for development
     purposes.
 
+OIDC_PROVIDER
+    If this value is provided, then OIDC is configured and SSL is used. It is
+    the URI of the OpenID Connect provider, which should return an OIDC
+    provider configuration document.
 
+SERVER_NAME
+    A standard flask configuration variable, but required if OIDC is configured
+    so the server can know its correct redirect URI.
+
+OIDC_CLIENT_ID, OIDC_CLIENT_SECRET
+    These are the client id and secret arranged with the OIDC provider,
+    if client registration is manual (google, for instance). If the provider
+    supports automated registration they are not required or used.
```

Comment:
We're going to have to have shared state anyway I think. You could limit keys to a single server instance and try and keep the clients 'sticky', but it doesn't work: You use the web browser to log in and set up a key, and that session sticks to one server; you use the key in the command line client, but that session goes to another server. You really need the servers to share the sessions.